openprescribing/api/views_spending.py

```diff
@@ -233,6 +234,10 @@ def price_per_unit(request, format=None):
         ON {ppusavings_table}.practice_id = {practice_table}.code
     LEFT OUTER JOIN {dmdproduct_table}
         ON {ppusavings_table}.bnf_code = {dmdproduct_table}.bnf_code
+    LEFT OUTER JOIN {dmdvmpp_table}
+        ON {dmdproduct_table}.vpid = {dmdvmpp_table}.vpid
+    LEFT OUTER JOIN {ncsoconcession_table}
+        ON {dmdvmpp_table}.vppid = {ncsoconcession_table}.vmpp_id
```

Comment:
Don't we also need to join on the date in question? Otherwise we'll get concession prices for all dates for any product?

Reply:
Good spot, thanks.